README.md

```diff
@@ -50,6 +50,6 @@ When the specification refers to a path in the context of an OCI layer tar (e.g.
 
 These documents currently specify:
 
-- Buildpack API: `0.6`
+- Buildpack API: `0.8`
```

Comment:
@natalieparellano I don't think we should be touching the Buildpack API in the `platform/0.9` branch.